### PR TITLE
Changed DMDbase.dynamics to handle negative eigenvalues

### PR DESCRIPTION
**Pull request by edonadini**
DMDbase.dynamics I removed the log in the computation of omega to handle negative eigenvalues.

### CODE REVIEW

**Comment by mtezzele** [comment]
@edonadini Could you please edit the description above with the actual new formula, instead of the code?

**Comment by edonadini** [comment]
DMDbase.dynamics I removed the log in the computation of omega to handle negative eigenvalues. 
My changes are based on [http://orlandi.ing.univaq.it/pub/Kutz%20J.%20Dynamic%20Mode%20Decomposition.%20Data-Driven%20Modeling%20of%20Complex%20Systems%202016.pdf](url) equation 1.24.
The new formula is in the attachment

image

**Comment by ndem0** [comment]
> DMDbase.dynamics I removed the log in the computation of omega to handle negative eigenvalues.
> My changes are based on [http://orlandi.ing.univaq.it/pub/Kutz%20J.%20Dynamic%20Mode%20Decomposition.%20Data-Driven%20Modeling%20of%20Complex%20Systems%202016.pdf](url) equation 1.24.
> The new formula is in the attachment
> 
> image

I would ask you to put the reference also in the documentation of the `dinamics` attribute.
P.S. please check the ref url, the current one is not valid.

File: pydmd/dmdbase.py [hunk 1]
```diff
@@ -118,15 +117,22 @@ def eigs(self):
     def dynamics(self):
         """
         Get the time evolution of each mode.
+        
+        .. math::
```
Comment (mtezzele) [inline pydmd/dmdbase.py:121]: Just a return here after "::" and we are done. Thanks

**Pull request merged**